clients/python-wrapper/tests/utests/common.py

```diff
@@ -0,0 +1,77 @@
+import importlib
+import os
+from contextlib import contextmanager
+from copy import deepcopy
+
+import lakefs_sdk
+import lakefs.repository
+from lakefs import config as client_config
+
+TEST_SERVER = "https://test_server"
+TEST_ACCESS_KEY_ID = "test_access_key_id"
+TEST_SECRET_ACCESS_KEY = "test_secret_access_key"
+TEST_CONFIG = f'''
+server:
+  endpoint_url: {TEST_SERVER}
+
+credentials:
+    access_key_id: {TEST_ACCESS_KEY_ID}
+    secret_access_key: {TEST_SECRET_ACCESS_KEY}
+'''
+
+TEST_CONFIG_KWARGS: dict[str, str] = {
```

Comment:
This is probably used for specific test suite to verify we process the configuration - so I think it should be moved from common py